server/WebAPI/appsettings.json

```diff
@@ -11,5 +11,15 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "EmailConfiguration": {
+    "From": "makemetripapp@gmail.com",
+    "SmtpServer": "smtp.gmail.com",
+    "Port": 587,
+    "Username": "makemetripapp@gmail.com",
```

Comment:
is it really mail address should be here?

Reply:
I think yes